utils/promisify.js

```diff
@@ -0,0 +1,12 @@
+const pify = require("pify");
+
+module.exports.sb = (input, ...args) => {
```

Comment:
Wow, this did simplify things.

Reply:
It's still a bit complicated since Sendbird doesn't use the standard error-first callbacks (I opened an issue for this, [Error-first callbacks](https://github.com/smilefam/SendBird-SDK-JavaScript/issues/93)). If that doesn't happen, we'll have to continue using this method until [Promises are added](https://github.com/smilefam/SendBird-SDK-JavaScript/issues/9). I can't even open a PR for either of the alternatives due to their no-PR policy. 🙁